frontend/views/containers/modals/CreateGroup.vue

```diff
@@ -191,6 +215,7 @@ export default {
       },
       incomeProvided: {
         required,
+        minValue: (val) => val > 0,
```

Comment:
Proper way to use `minValue` is to import `minValue` from `from 'vuelidate/lib/validators'` and then use it here like so:

```
minValue: minValue(0)
```

Reply:
It doesn't find minValue in the library

Reply:
Really? You tried importing it `from 'vuelidate/lib/validators'`?

If so, then this is OK.